crates/bevy_render/src/mesh/mesh/mod.rs

```diff
@@ -1183,6 +1218,15 @@ where
     }
 }
 
+#[derive(Debug, Error)]
+pub enum MeshWindingInvertError {
+    #[error("Source mesh does not have primitive topology TriangleList")]
+    WrongTopology,
```

Comment:
We should report the actual topology here.

Reply:
Could do that but other functions like `triangles()` just error like this, while others panic directly. And it already says what it wants.